serverless-functions/src/functions/features/callback-and-voicemail/common/callback-operations.private.js

```diff
@@ -23,37 +23,51 @@ exports.createCallbackTask = async (parameters) => {
   } = parameters;
 
   // use assigned values or use defaults
-  const workflowSid = overriddenWorkflowSid || process.env.TWILIO_FLEX_CALLBACK_WORKFLOW_SID;
-  const timeout = overriddenTimeout || 86400;
-  const priority = overriddenPriority || 0;
-  const attempts = retryAttempt || 0;
-  const taskChannel = overriddenTaskChannel || 'voice';
+  const workflowSid =
+    overriddenWorkflowSid || originalTask?.workflowSid || process.env.TWILIO_FLEX_CALLBACK_WORKFLOW_SID;
+  const timeout = overriddenTimeout || originalTask?.timeout || 86400;
+  const priority = overriddenPriority || originalTask?.priority || 0;
+  const attempts = retryAttempt || originalTask?.attributes?.callBackData?.attempts + 1 || 0;
+  const taskChannel = overriddenTaskChannel || originalTask?.taskChannelUniqueName || 'voice';
+  const taskType = recordingSid || originalTask?.attributes?.callBackData?.recordingSid ? 'voicemail' : 'callback';
```

Comment:
@dremin - please ignore my previous comment. For reability only, it might make sense to change it 
`const taskType = (recordingSid || originalTask?.attributes?.callBackData?.recordingSid) ? 'voicemail' : 'callback';`